packages/react-native-gesture-handler/android/spotless.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: "com.diffplug.spotless"
 
 spotless {
     kotlin {
-        target "src/**/*.kt", "reanimated/**/*.kt", "noreanimated/**/*.kt", "svg/**/*.kt", "nosvg/**/*.kt", "common/**/*.kt"
+        target "**/*.kt"
```

Comment:
Since we're changing basically everything, can you check if updating spotless causes any problems?

Reply:
Well, we can use newer `spotless` 😅 But since this PR is already quite large I've done it in #3507